validation/kubernetes/velero/main.tf

```diff
@@ -1,12 +1,20 @@
 terraform {
```

Comment:
Should this file be changed in this PR

Reply:
Not really, but fixed it since I saw that I've missed it. ;)